src/app/api/fetchSearch.ts

```diff
@@ -0,0 +1,17 @@
+import axios from "axios";
+import { API_KEY, API_URL, ResponseAPI } from "./constants";
+
+export async function fetchSearch({
+  request,
+}: {
+  request: string;
+}): Promise<ResponseAPI> {
+  try {
+    const url = `${API_URL}/everything?q=${request}&pageSize=20&apiKey=${API_KEY}`;
+    const { data } = await axios<ResponseAPI>(url);
+    return data;
+  } catch (err) {
```

Comment:
What is the point in catching an error, do nothing and then throw the same error? If you handle this error in a different place, please remove try catch